autoload/rainbow_csv.vim

```diff
@@ -820,12 +818,13 @@ func! rainbow_csv#align_field(field, is_first_line, max_field_components_lens, i
     return repeat(' ', integer_delta_length) . clean_field . trailing_spaces
 endfunc
 
-
-func! rainbow_csv#csv_align()
+func! rainbow_csv#csv_align() range
+    let l:firstline = a:firstline
+    let l:lastline = a:lastline
     " The first (statistic) pass of the function takes about 40% of runtime, the second (actual align) pass around 60% of runtime.
     " Numeric-aware logic by itself adds about 50% runtime compared to the basic string-based field width alignment
     " If there are lot of numeric columns this can additionally increase runtime by another 50% or more.
-    let show_progress_bar = wordcount()['bytes'] > 200000
+    let show_progress_bar = (l:lastline - l:firstline) > 200000
```

Comment:
Please revert, you replaced byte count with line count

Reply:
done